src/common-principles.md

```diff
@@ -196,6 +196,18 @@ as the labels would collide on a case-insensitive filesystem.
 Additionally, because the suffix `eeg` is defined,
 then the suffix `EEG` will not be added to future versions of the standard.
 
+## No duplicate files under different file extensions
+
+BIDS datasets MUST NOT contain identical data under different extensions.
```

Comment:
```suggestion
BIDS datasets MUST NOT contain identical data under different extensions.
```

I think that in the long term we may to be clearer as to what it meant by "identical data" but any clarification could be done in subsequent PR.

Reply:
Actually, let's forget "identical data". Instead we should avoid identical file base names with identical extensions. We don't care whether data are "identical" or not.

Reply:
I assume you are not including the json sidecar in this case. :-p

More seriously, you got a bunch of cases where data must come in pairs of files (sometimes even 3) where the files only differ by extension.

take eeg for example:

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#eeg-recording-data

- .vhdr, .vmrk, .eeg
- .set, .fdt

Hence the emphasis on the "data" file.

Reply:
You're right of course 🤐 Then I guess "identical data", "equivalent data" or "interchangeable data" are good enough for now.